### PR TITLE
Refactor/241 remove lobobrowser AND fix classpath issues

### LINKED ISSUE #241: Replace LoboBrowser components
**Component to be refactored**
LoboBrowser code

**Purpose of refactoring**
I just spent about 3 hours trying to figure out why a method didn't exist during compile and determine that LoboBrowser embedded the class files of one of it's dependencies instead of just depending on them. And since java is first class loaded wins, I couldn't use the newer version of the library I desired.

This is the 2nd library that has bit us with Lobo. the first was JOOQ and this time it was slf4j. 

**Proposed method**
There are certainly other HTML renders available which is primarily what it is used for. There is some usage in the WebDirectoryDataSource however that can likely be replaced simple some java 8 built-ins.

**Additional context**
yes I'm mostly frustrated, but the packaging is extremely sloppy so why would we want to depend on such.

### PR DESCRIPTION
**Pull request by MikeNeilson**
## Problem Description

Fixes #241. 
Fixes issue with large classpaths by using wild card class path for the two directories.
Fixes issue with determined vs set DCSTOOL_HOME on POSIX system reporting an a different when there really isn't one.

## Solution

Moved block in WebDirectoryDataSource to use simple regex match to find link, verify that at least for my understanding the behavior is correct (I think the original DataSource folder structure has changed.)
Used wildcard classpaths.
Compared determined "realpath" to determine if environment setup is questionable.

Removed LoboBrowser and Cobra dependencies and references thereto.

## how you tested the change
On windows/linux

Verified applications could run from ant run task, shell/cmd, or double clicking the .bat file in windows.

## Where the following done:

- [ ] Tests. Check all that apply:
   - [ ] Unit tests created or modified that run during ant test.
   - [ ] Integration tests created or modified that run during integration testing
         (Formerly called regression tests.)
   - [ ] Test procedure descriptions for manual testing
- [ ] Was relevant documentation updated?
- [ ] Were relevant config element (e.g. XML data) updated as appropriate

If you aren't sure leave unchecked and we will help guide you to want needs changing where.

### CODE REVIEW

**Comment by MikeNeilson** [comment]
Okay, this time I think I've got it.

**Comment by ktarbet** [comment]
I did some basic testing in windows and Linux.   I verified there are two less dependencies in the ./dep directory in a install.:

These jars are gone from the install in this branch, but exist in the master branch.

```bash
./OPENDCS.master/dep/zzzCobra-1.0.2.jar
./OPENDCS.master/dep/zzzLoboBrowser-1.0.0.jar
```

File: install/bin/decj [hunk 1]
```diff
@@ -50,43 +50,9 @@ fi
...
-# then
-#   CP=$CP:$CWMS_JAR_DIR/hec.jar:$CWMS_JAR_DIR/hecData.jar:$CWMS_JAR_DIR/heclib.jar:$CWMS_JAR_DIR/rma.jar:$CWMS_JAR_DIR/lookup.jar
-# fi
-# if [ -n "$CWMS_SYSJAR_DIR" -a -d "$CWMS_SYSJAR_DIR" ]
-# then
-#   CP=$CP:$CWMS_SYSJAR_DIR/jdom.jar:$CWMS_SYSJAR_DIR/jdom-1.0.jar
-# fi
-
-
-if [ ! -z "$CP_SHARED_JAR_DIR" -a -d "$CP_SHARED_JAR_DIR" ]
```
Comment (MikeNeilson) [inline install/bin/decj:64]: @agilmore2 where these for HDB, the ones above that says CWMS are rather obvious and we don't need them anymore but I'm not sure if I went to far with these.

**Review by ktarbet — APPROVED** [review]
The website referenced in the WebDirectoryDataSource dd.weatheroffice.gc.ca - doesn't seem to be active so testing/issue-feedback is being left to anyone using WebDirectoryDataSource.

**Comment by MikeNeilson** [comment]
I'll update the source with the link I used to at least somewhat test it, but I think the general structure changed and this should operate recursively, but I'm going to wait for someone to ask before doing the work.

**Comment by agilmore2** [comment]
I don't see usage of this $CP_SHARED_JAR_DIR in any of the HDB instances,
but it is a documented feature:

https://opendcs-env.readthedocs.io/en/latest/legacy-cp-userguide.html#computation-meta-data

I do like the ability to drop a library of algorithms in a directory and
have them work.

**Comment by MikeNeilson** [comment]
Hmm. Thanks i'll go read the docs. But that behavior is already covered by
the userdir, wouldnt be surprised if this was for those that didnt use it.
But that at least meams it's not the right time to yank it out without
warning.

**Review by ktarbet — APPROVED** [review]

**Pull request merged**